README.md
# AdminThemes

admin-blankon/changelog.txt
In this file I will explain more detail about change log Blankon version. Enjoy cheers! :D

Update 1.1.10 (-) - Latest
----------
 - NEW: Blankon special (Investor)
 - NEW: Blankon Codeigniter (pure framework) version
 - IMPROVEMENT: Blankon Codeigniter+Smarty (add all files framework CI)
 - UPDATED: Update documentation page (Codeigniter)

Update 1.1.9 (29/09/2016)
----------
 - NEW: Blankon special (Human Resources)
 - UPDATED: Cubeportfolio update version from 3.7.0 to 3.8.0 (Latest version)

Update 1.1.8 (24/08/2016)
----------
 - NEW: Blankon special (E-Commerce System)
 - NEW: Blankon special (Project Management) add project design
 - UPDATED: Update version MDBootstrap plugin to v3.4.1 (latest version)
 - UPDATED: Update documentation page (HTML) add menu Blankon special

Update 1.1.7 (12/07/2016)
----------
 - FIX: Decompression issues zip file

Update 1.1.6 (07/07/2016)
----------
 - NEW: Blankon special (Project Management)
 - NEW: Frontend themes (Revolution Slider)
 - UPDATED: Cubeportfolio update version from 3.6.0 to 3.7.0
 - UPDATED: Slider Revolution update version from 5.1.6 to 5.2.0
 - IMPROVEMENT: Add feature choose preloader frontend themes on documentation page

Update 1.1.5 (13/03/2016)
----------
 - NEW: Material design version
 - NEW: One page revolution slider (photography type 2)
 - NEW: UI features notifications page
 - IMPROVEMENT: Framework for material design FREE & PRO (mdbootstrap.com) (save $49)

Update 1.1.4 (15/02/2016)
----------
 - NEW: One page revolution slider (gym)
 - NEW: One page revolution slider (not generic)
 - NEW: One page revolution slider (fashion)
 - UPDATED: Update version cubeportfolio plugin to v3.5.2 (latest version)

Update 1.1.3 (10/02/2016)
----------
 - NEW: Datatable with unlimited features
 - NEW: One page revolution slider (web product light)
 - NEW: One page revolution slider (web product dark)
 - NEW: One page revolution slider (photography)
 - NEW: One page revolution slider (restaurant)
 - IMPROVEMENT: Add new search system design for implementation on project
 - IMPROVEMENT: Add rating popular github on third party plugins
 - UPDATED: Update version cubeportfolio plugin to v3.4.3 (latest version)
 - UPDATED: Update version revolution slider plugin to v5.1.6 (latest version)

Update 1.1.2 (13/11/2015)
----------
 - NEW: App showcase one page frontend theme
 - NEW: Dashboard human resources (HR) design
 - NEW: Dashboard retail analysis design
 - NEW: Map vector type 2 design
 - IMPROVEMENT: Add tour guide on dashboard basic design
 - IMPROVEMENT: Include retina display to all version
 - IMPROVEMENT: Add partial plugins SASS & LESS
 - IMPROVEMENT: Change UI/UX design on all documentation file
 - IMPROVEMENT: Add panel tabs


Update 1.1.1 (20/09/2015)
----------
 - NEW: Dashboard marketing campaign design
 - NEW: Dashboard real estate design
 - NEW: Project team board
 - NEW: Project team profile
 - NEW: Project clients
 - NEW: Component list group basic
 - NEW: Component list group samples
 - NEW: UI Feature Bootstrap tour
 - NEW: UI Feature grid stack
 - IMPROVEMENT: Form picker (add date range picker)
 - IMPROVEMENT: Blog grid type 2 (add search system)
 - IMPROVEMENT: Widget social (add overview social facebook, twitter, youtube)
 - IMPROVEMENT: Widget overview (add mini stat type 5)

Update 1.1.0 (03/09/2015)
----------
 - NEW: Dashboard investor design
 - NEW: Dashboard projects design
 - NEW: Easy pie chart page
 - NEW: Component flag icon CSS
 - NEW: Form picker
 - NEW: Project issue tracker
 - NEW: Chart chartist
 - IMPROVEMENT: Component tabs & accordions (add border color on tabs)
 - IMPROVEMENT: Component modals (add modal form layout)
 - IMPROVEMENT: Widget overview (add mini stat type 4)
 - IMPROVEMENT: Table samples (add issue list table)

Update 1.0.10 (26/08/2015)
----------
 - NEW: E-Commerce dashboard design
 - NEW: Gallery page type 3
 - NEW: Gallery page type 4
 - NEW: Project portfolio design type 4
 - NEW: Peity charts page
 - NEW: Table samples page
 - NEW: Timeline page type 2
 - IMPROVEMENT: Table color page
 - IMPROVEMENT: Component other (label, badge, progress bar)
 - IMPROVEMENT: Widget overview (add mini stats type 2)
 - IMPROVEMENT: Widget misc (add recent activity)
 - UPDATED: Bootstrap changed version to 3.3.5
 - UPDATED: Font Awesome changed version to 4.4.0

Update 1.0.9 (17/08/2015)
----------
 - NEW: Blog post design type 2
 - NEW: Project team design
 - NEW: Project portfolio design
 - NEW: Project portfolio design type 2
 - NEW: Project portfolio design type 3
 - NEW: Gallery page type 2
 - NEW: Search page basic
 - NEW: Mail templates
 - NEW: Mail responsive layout
 - IMPROVEMENT: Component others
 - IMPROVEMENT: Component Tabs & Accordions (add tabs animation)

Update 1.0.8 (07/02/2015)
----------
 - NEW: Blankon Yeoman + GruntJS
 - NEW: Initial documentation for Yeoman version
 - UPDATED: Documentation page on HTML version
 - IMPROVEMENT: Preprocessor CSS using SASS + Compass
 - FIXED: Bug fixing on lost password AngularJS version

Update 1.0.7 (09/05/2015)
----------
 - NEW: Blankon Codeigniter + Smarty (template engine) version
 - NEW: FAQ page design
 - NEW: Animate.css page design
 - IMPROVEMENT: Add Slider Revolution plugin commercial
 - IMPROVEMENT: Add Cube Portfolio plugin commercial
 - FIXED: Bug fixing on sidebar left style on all versions

Update 1.0.6 (23/04/2015)
----------

 - NEW: Blankon PHP version
 - NEW: Login style type 2 design
 - NEW: Video page
 - UPDATED: Bootstrap Library to v3.3.4
 - IMPROVEMENT: Add installation guide Yii2 version
 - IMPROVEMENT: Add installation guide Laravel version
 - IMPROVEMENT: Logic JS Blankon AJAX version
 - IMPROVEMENT: Simple line icons component
 - FIXED: Bug fixing on responsive style
 - FIXED: Bug fixing on Blankon AJAX version

Update 1.0.5 (07/04/2015)
----------

admin-blankon/production/admin/php/view/layout.php
<!DOCTYPE html>
<!--[if IE 8]> <html lang="en" class="ie8"> <![endif]-->
<!--[if IE 9]> <html lang="en" class="ie9"> <![endif]-->
<!--[if !IE]><!--> <html lang="en"> <!--<![endif]-->

<!-- START @HEAD -->
<head>
    <!-- START @META SECTION -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="description" content="{description}">
    <meta name="keywords" content="{keywords}">
    <meta name="author" content="Djava UI">
    <title>{title}</title>
    <!--/ END META SECTION -->

    <!-- START @FAVICONS -->
    <link href="http://themes.djavaui.com/blankon-fullpack-admin-theme/img/ico/html/apple-touch-icon-144x144-precomposed.png" rel="apple-touch-icon-precomposed" sizes="144x144">
    <link href="http://themes.djavaui.com/blankon-fullpack-admin-theme/img/ico/html/apple-touch-icon-114x114-precomposed.png" rel="apple-touch-icon-precomposed" sizes="114x114">
    <link href="http://themes.djavaui.com/blankon-fullpack-admin-theme/img/ico/html/apple-touch-icon-72x72-precomposed.png" rel="apple-touch-icon-precomposed" sizes="72x72">
    <link href="http://themes.djavaui.com/blankon-fullpack-admin-theme/img/ico/html/apple-touch-icon-57x57-precomposed.png" rel="apple-touch-icon-precomposed">
    <link href="http://themes.djavaui.com/blankon-fullpack-admin-theme/img/ico/html/apple-touch-icon.png" rel="shortcut icon">
    <!--/ END FAVICONS -->

    <!-- START @FONT STYLES -->
    <link href="http://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700" rel="stylesheet">
    <link href="http://fonts.googleapis.com/css?family=Oswald:700,400" rel="stylesheet">
    <!--/ END FONT STYLES -->

    <!-- START @GLOBAL MANDATORY STYLES -->
    <link href="{url}assets/global/plugins/bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
    <!--/ END GLOBAL MANDATORY STYLES -->

    <!-- START @PAGE LEVEL STYLES -->
    {css}
    <!--/ END PAGE LEVEL STYLES -->

    <!-- START @THEME STYLES -->
   {cssTheme}
    <!--/ END THEME STYLES -->

    <!-- START @IE SUPPORT -->
    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
    <script src="{url}assets/global/plugins/bower_components/html5shiv/dist/html5shiv.min.js"></script>
    <script src="{url}assets/global/plugins/bower_components/respond-minmax/dest/respond.min.js"></script>
    <![endif]-->
    <!--/ END IE SUPPORT -->
</head>
<!--/ END HEAD -->

<!--

|=========================================================================================================================|
|  TABLE OF CONTENTS (Use search to find needed section)                                                                  |
|=========================================================================================================================|
|  01. @HEAD                        |  Container for all the head elements                                                |
|  02. @META SECTION                |  The meta tag provides metadata about the HTML document                             |
|  03. @FAVICONS                    |  Short for favorite icon, shortcut icon, website icon, tab icon or bookmark icon    |
|  04. @FONT STYLES                 |  Font from google fonts                                                             |
|  05. @GLOBAL MANDATORY STYLES     |  The main 3rd party plugins css file                                                |
|  06. @PAGE LEVEL STYLES           |  Specific 3rd party plugins css file                                                |
|  07. @THEME STYLES                |  The main theme css file                                                            |
|  08. @IE SUPPORT                  |  IE support of HTML5 elements and media queries                                     |
|=========================================================================================================================|
|  09. @BODY                        |  Contains all the contents of an HTML document                                      |
|  10. @WRAPPER                     |  Wrapping page section                                                              |
|  11. @HEADER                      |  Header page section contains about logo, top navigation, notification menu         |
|  12. @SIDEBAR LEFT                |  Sidebar page section contains all sidebar menu left                                |
|  13. @PAGE CONTENT                |  Contents page section contains breadcrumb, content page, footer page               |
|  14. @SIDEBAR RIGHT               |  Sidebar page section contains all sidebar menu right                               |
|  15. @BACK TOP                    |  Element back to top and action                                                     |
|=========================================================================================================================|
|  16. @CORE PLUGINS                |  The main 3rd party plugins script file                                             |
|  17. @PAGE LEVEL PLUGINS          |  Specific 3rd party plugins script file                                             |
|  18. @PAGE LEVEL SCRIPTS          |  The main theme script file                                                         |
|=========================================================================================================================|

START @BODY
|=========================================================================================================================|
|  TABLE OF CONTENTS (Apply to body class)                                                                                |
|=========================================================================================================================|
|  01. page-boxed                   |  Page into the box is not full width screen                                         |
|  02. page-header-fixed            |  Header element become fixed position                                               |
|  03. page-sidebar-fixed           |  Sidebar element become fixed position with scroll support                          |
|  04. page-sidebar-minimize        |  Sidebar element become minimize style width sidebar                                |
|  05. page-footer-fixed            |  Footer element become fixed position with scroll support on page content           |
|  06. page-sound                   |  For playing sounds on user actions and page events                                 |
|=========================================================================================================================|

-->
<body class="{body}">

<!--[if lt IE 9]>
<p class="upgrade-browser">Upps!! You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/" target="_blank">upgrade your browser</a> to improve your experience.</p>
<![endif]-->

<!-- START @WRAPPER -->
<section id="wrapper">

<!-- START @HEADER -->
{header} <!-- /#header -->
<!--/ END HEADER -->

<!--

START @SIDEBAR LEFT
|=========================================================================================================================|
|  TABLE OF CONTENTS (Apply to sidebar left class)                                                                        |
|=========================================================================================================================|
|  01. sidebar-box               |  Variant style sidebar left with box icon                                              |
|  02. sidebar-rounded           |  Variant style sidebar left with rounded icon                                          |
|  03. sidebar-circle            |  Variant style sidebar left with circle icon                                           |
|=========================================================================================================================|

-->
{sidebarLeft}<!-- /#sidebar-left -->
<!--/ END SIDEBAR LEFT -->

<!-- START @PAGE CONTENT -->
<section id="page-content">

{content}<!-- /.body-content -->


<!-- Start footer content -->
{footer}<!-- /.footer-content -->
<!--/ End footer content -->

</section><!-- /#page-content -->
<!--/ END PAGE CONTENT -->

<!-- START @SIDEBAR RIGHT -->
{sidebarRight}<!-- /#sidebar-right -->
<!--/ END SIDEBAR RIGHT -->

</section><!-- /#wrapper -->
<!--/ END WRAPPER -->

<!-- START @BACK TOP -->
<div id="back-top" class="animated pulse circle">
    <i class="fa fa-angle-up"></i>
</div><!-- /#back-top -->
<!--/ END BACK TOP -->

<!-- START JAVASCRIPT SECTION (Load javascripts at bottom to reduce load time) -->
<!-- START @CORE PLUGINS -->
{corePlugins}
<!--/ END CORE PLUGINS -->

<!-- START @PAGE LEVEL PLUGINS -->
{levelPlugins}
<!--/ END PAGE LEVEL PLUGINS -->

<!-- START @PAGE LEVEL SCRIPTS -->
{levelScript}
<!--/ END PAGE LEVEL SCRIPTS -->
<!--/ END JAVASCRIPT SECTION -->

<!-- START GOOGLE ANALYTICS -->
<script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-55892530-1', 'auto');
    ga('send', 'pageview');

</script>
<!--/ END GOOGLE ANALYTICS -->

</body>
<!--/ END BODY -->

</html>